src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -934,7 +934,9 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                     Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                     TestHelper.VerifyRequestMethod(response, method);
                     string responseContent = await response.Content.ReadAsStringAsync();
-                    _output.WriteLine(responseContent);                    
+                    _output.WriteLine(responseContent);
+
+                    Assert.Contains($"\"Content-Length\": \"{request.Content.Headers.ContentLength.Value}\"", responseContent);
```

Comment:
Why add this?  Is this to verify that the request was sent using 'Content-Length' semantics and no "Transfer-Encoding: chunked" semantics?